donner.go

```diff
@@ -105,15 +106,21 @@ func execCommand(cfg *Cfg, params []string) error {
 	}
 
 	cmd := exec.Command(execHandler.BaseCommand, cliArgs...)
-
-	out, err := cmd.CombinedOutput()
-	if err != nil {
-		return err
+	cmd.Stdout = os.Stdout
+	cmd.Stdin = os.Stdin
+	cmd.Stderr = os.Stderr
+
+	var waitStatus syscall.WaitStatus
+	if err := cmd.Run(); err != nil {
```

Comment:
This whole block handles only the error case which we technically don't need because docker is logging to `stderr` which you got covered by assigning `cmd.Stderr = os.Stderr`. The only case you're handling with the waitstatus is when the command can't be executed which could be as simple as L115.